GVFS/GVFS.Platform.Mac/MacFileSystem.cs

```diff
@@ -32,20 +32,21 @@ public override bool IsSocket(string fileName)
 
         public override bool IsFileSystemSupported(string path, out string error)
         {
-            string lowerCaseFile = Guid.NewGuid().ToString().ToLower();
-            string upperCaseFile = lowerCaseFile.ToUpper();
             error = null;
 
             try
             {
-                File.Create(Path.Combine(path, lowerCaseFile));
-                if (File.Exists(Path.Combine(path, upperCaseFile)))
+                string lowerCaseFilePath = Path.Combine(path, Guid.NewGuid().ToString().ToLower());

```

Comment:
An aside: what are the chances that a GUID has no letters?

  10^32 / 16^32 = (5/8)^32 &approx;  1.7067^(-68)

OK. This is reasonable.

Reply:
That's a good point, it would be more robust to name the file something like `$"casetest{GUID}"`

Any objections to make that change as part of this?

Reply:
Sounds like a simple change. Thanks!